### PR TITLE
Vertical Timeline Component is now properly displaying dots for all m…

### PR DESCRIPTION
**Pull request by devin-mitchell**
…edia queriesimage
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] `npm test`

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [x] My changes generate no new warnings
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by cooper-jackson — APPROVED** [review]
Worked on this with Devin -- Displays dots correctly as well as consolidates stylesheets for the timeline react component to a single file, rather than multiple with confusing naming conventions.

**Comment by rhiag** [comment]
Looks good!

File: src/components/timeline/TimelineItems.css [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+    100% {
+      -webkit-transform: scale(1);
+      -moz-transform: scale(1);
+      -ms-transform: scale(1);
+      -o-transform: scale(1);
+      transform: scale(1);
+    }
+  }
+  
+  @media only screen and (max-width: 768px) {
```
Comment (alvin-joseph) [inline src/components/timeline/TimelineItems.css:192]: Nice work accounting for different screen widths!

**Review by alvin-joseph — APPROVED** [review]
Everything looks great! Nice work!

**Review by RyderBlues — APPROVED** [review]
Looks great bud 👍

**Comment by RyanFikejs** [comment]
Great guys, please check the appropriate boxes beneath your screen shot and proceed with your merge!

**Pull request merged**